src/services/orderService.ts

```diff
@@ -24,8 +23,8 @@ const getById = async (id: number): Promise<OrderDto> => {
   return await response.json();
 };
 
-const create = async (createDto: OrderCreateDto): Promise<OrderDto> => {
-  const response = await httpClient.post('/orders', createDto);
+const create = async (): Promise<OrderDto> => {
+  const response = await httpClient.post('/orders', {});
```

Comment:
a post to order accepts a body of OrderDto containing a collection of LineItems (from checking the api)